doc/source/users_guide/cime-change-namelist.rst

```diff
@@ -3,24 +3,29 @@
 Customizing your input variables
 ================================
 
-CIME and current CIME-compliant components currently primarily uses fortran namelists.
+CIME and CIME-compliant components currently use Fortran namelists to control runtime options.
 
-All CIME-compliant components generate their input variable settings using a **buildnml** file located in the component's **cime_config** directory.
+All CIME-compliant components generate their namelist file using a **buildnml** script located in the component's **cime_config** directory.  **buildnml** may call other scripts to complete the namelist.
```

Comment:
the buildnml is usually located in the components cime_config directory but this is actually controlled by config_files.xml

Reply:
Will note that.